scripts/create_cita_config.py

```diff
@@ -449,6 +463,24 @@ def parse_arguments():
         help='The address of new node. Will generate a new address (with privkey) if not set.'
     )
 
+    # enable encrypted
+    pappend.add_argument(
+        '--enable_tls',
+        action='store_true',
+        help='The data is encrypted and transmitted on the network')
+
+    # enable get version
+    pappend.add_argument(
+        '--enable_version',
+        action='store_true',
+        help='Jsonrpc will return cita version')
+
+    # switch to stdout
+    pappend.add_argument(
+        '--stdout',
+        action='store_true',
```

Comment:
Default value is true while appending a new node ?

Reply:
This `action='store_true'` does not mean its take `true` as default.
You can read 
https://docs.python.org/zh-cn/3/howto/argparse.html
for more detail about `ArgumentParser`.

Reply:
such a awkward usage.